build.gradle

```diff
@@ -109,8 +109,8 @@ subprojects {
 		googleCloudVersion = '2.0.5'
 		googleTraceVersion = '2.0.0'
 		cloudMonitoringVersion = '3.0.0'
-		openTelemetryVersion = '1.10.0'
-		openTelemetryInstrumentationVersion = '1.10.0'
+		openTelemetryVersion = '1.12.0'
```

Comment:
Ah, checking the tagged released of the auto jar, I think it was released for `1.11.0` so will need to chagne this before merge